File: snippets/contact-center/create-workspace/create-workspace.js
const accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
const authToken = 'your_auth_token';
const client = require('twilio')(accountSid, authToken);

client.taskrouter.workspaces
                 .create({
                    template: 'FIFO',
                    friendlyName: 'NewWorkspace'
                  })
                 .then(workspace => console.log(workspace.sid))
                 .done();

File: snippets/contact-center/create-workspace/create-workspace.cs
using System;
using Twilio;
using Twilio.Rest.Taskrouter.V1;

class Program
{
  static void Main(string[] args)
  {
    // Find your Account Sid and Token at twilio.com/console
    const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
    const string authToken = "your_auth_token";

    TwilioClient.Init(accountSid, authToken);

    var workspace = WorkspaceResource.Create(
        template: "FIFO",
        friendlyName: "NewWorkspace"
    );

    Console.WriteLine(workspace.Sid);
  }
}
